src/components/index.js

```diff
@@ -26,7 +26,12 @@ class App extends Component {
 
   componentWillMount() {
     this.hostname = document.location.hostname;
-    this.isProduction = !!this.hostname.match(/^(www|mobile).abc.net.au$/);
+    let isPl = !!this.hostname.match(
+      /.*presentation-layer\.abc-(prod|test)\.net\.au$/
+    );
+    if (isPl) isPl = !this.hostname.match(/-preview/);
+    const isProduction = !!this.hostname.match(/^(www|mobile).abc.net.au$/);
+    this.isProduction = isProduction || isPl;
```

Comment:
This is quite a confusing block of code. If I've read it correctly, any PL URL that also includes `-preview` will make `this.isProduction ===  false`. Could we make it a positive match against PL production domains instead?

I'd also probably prefer to exclude testing domains from the database. My read of this is that `presentation-layer.abc-test.net.au` would be included?

Reply:
Yeah that was the goal, not the most beautiful thing.

To your second point, if you'd prefer any testing domains not included. I've removed the PL check and amended the original regex to check for subdomains of abc.net.au.

Reply:
Unfortunately that doesn't work either because preview is a subdomain: `nucwed.aus.aunty.abc.net.au`. The production subdomains need to be explicitly tested for.

Reply:
This should only match first level subdomains of abc.net.au, which would rule out nucwed (unless there are others).

Reply:
I don't know if there are others—things change in unexpected and often unannounced ways, which is why I like to play things like this safe.

What domains do you actually want to match as production? Can we just list them?

Reply:
Fair enough, I've updated to `/^(www|mobile|rocketclock|rocketclock-staging|bigted|bigted-staging).abc.net.au$/`